3d aproksymacja/Locate bark.ipynb

```diff
@@ -79,13 +79,12 @@
     "import numpy as np\n",
     "import matplotlib.pyplot as plt\n",
     "from mpl_toolkits.mplot3d import Axes3D\n",
-    "n = points.shape[0]\n",
-    "nu = points1.shape[0]\n",
-    "nub = points2.shape[0]\n",
     "points = np.array(points).T\n",
     "points1 = np.array(points1).T\n",
     "points2 = np.array(points2).T\n",
-    "print(points.shape)\n",
+    "n = points.shape[1]\n",
+    "nu = points1.shape[1]\n",
+    "nub = points2.shape[1]\n",
```

Comment:
what are those variable names

Reply:
Could you please remove the comments in Polish, thank you in advance for resolving the issue quickly

Reply:
Could you please not use magic numbers. For an overview, I provide the following link:

https://en.wikipedia.org/wiki/Magic_number_(programming)#:~:text=In%20computer%20programming%2C%20a%20magic,see%20List%20of%20file%20signatures

Reply:
![](https://media.tenor.com/rWC7FaiwzZQAAAAM/what-are-those-dave-chappelle.gif)